.travis.yml

```diff
@@ -15,7 +15,7 @@ addons:
 
 script:
   - Rscript -e "tinytex::install_tinytex()"
-  - make everything
+  - make r-rse py-rse
```

Comment:
@gvwilson I have a fix as per my comment in #295, if you can revert this change, we can merge this edit it.

Reply:
Change reverted. Please try to get a fix for the data file in place before Tuesday's meeting - thanks.